src/features/surveys/managers/surveyAnswerManager.ts

```diff
@@ -63,12 +63,12 @@ export const useSurveyAnswerManager = (initialData: SurveyWithQuestions) => {
 
   const getSurveyData = useCallback(async () => {
     if (!initialData.isActive) {
-      router.replace('/');
+      // router.replace('/');
       return;
```

Comment:
It would be nice to:
1. Remove this if statement
2. In `src\pages\survey\[surveyId]\index.tsx` in `getServerSideProps` check if survey is active and if not create new surveyNotFound page and redirect to that.

Reply:
When the active survey is no longer active then we are displaying this message "Oops Survey is no longer active".
Isn't that enough?